_articles/assets/adding-and-playing-custom-sounds.md

```diff
@@ -1,9 +1,8 @@
 ---
 title: Adding and playing Custom Sounds
 author: Noya
-steamId: 76561198046984233
+steamId: '76561198046984233'
 date: 10.03.2015
```

Comment:
Could the date (and optionally the author) be moved to the top of the article to make it more obvious which are recent and which are out-of-date?